cluster/addons/dashboard/dashboard-controller.yaml

```diff
@@ -1,27 +1,29 @@
+{% set version = "0.1.0-alpha1" -%}
```

Comment:
I see other configs that have variables and still have *.yaml extension. What's the practice?